File: sdks/go/windjammer/time.go
package windjammer

import "fmt"

// Time represents time information for the current frame
type Time struct {
	DeltaSeconds float32 // Time since last frame in seconds
	TotalSeconds float32 // Total time since application start in seconds
	FrameCount   int     // Current frame number
}

// String returns a string representation of Time
func (t *Time) String() string {
	return fmt.Sprintf("Time(delta=%.3f, total=%.3f)", t.DeltaSeconds, t.TotalSeconds)
}

File: sdks/python/windjammer_sdk/time.py
"""Time utilities."""


class Time:
    """Time information for the current frame."""
    
    def __init__(self):
        self.delta_seconds = 0.016  # ~60 FPS
        self.total_seconds = 0.0
        self.frame_count = 0
    
    def __repr__(self) -> str:
        return f"Time(delta={self.delta_seconds}, total={self.total_seconds})"

